src/Microsoft.VisualStudio.Threading.Analyzers.Tests/VSTHRD003UseJtfRunAsyncAnalyzerTests.cs

```diff
@@ -90,6 +90,90 @@ public void Test()
         }
 
         [Fact]
+        public void ReportWarningWhenTaskTIsReturnedDirectlyFromLambda()
+        {
+            var test = @"
```

Comment:
I prefer the repro code to appear within the test method. Otherwise it becomes quite difficult to understand looking at a test method what it is actually testing.